keras_nlp/utils/preset_utils.py

```diff
@@ -109,6 +123,9 @@ def save_to_preset(
     weights_filename="model.weights.h5",
 ):
     """Save a KerasNLP layer to a preset directory."""
+    push_to_hf = preset.startswith(HF_PREFIX)
```

Comment:
Here we might end up doing some tweaks. We want to support a split flow for saving preprocessing and model weights. So our final flow might need to allow something like this

```python
tokenizer.save_to_preset(dir)
backbone.save_to_preset(dir)
upload_preset("hf://user/model", dir)
```

I don't think we need to solve that here though! @SamanehSaadat is working on a draft of our upload flow currently.

Reply:
Makes perfect sense!